modules/data-entry/src/main/frontend/src/questionnaireEditor/CreationMenu.jsx

```diff
@@ -31,7 +31,7 @@ import NewItemButton from "../components/NewItemButton";
 // Menu for creating questions or sections
 
 let CreationMenu = (props) => {
-  const { isMainAction, data, menuItems, models, onCreated } = props;
+  const { isMainAction = false, data, menuItems, models, onCreated } = props;
```

Comment:
No need for the default value `false`.
```suggestion
  const {
    isMainAction,
    data,
    menuItems,
    models,
    onCreated
  } = props;
```